microshift_updating/microshift-about-updates.adoc

```diff
@@ -6,11 +6,11 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.16 release, the following updates are supported:
+Updates are supported on {product-title} beginning with the General Availability version 4.14. With the 4.17 release, the following updates are supported:
 
 * A maximum of two minor versions to the next in sequence, for example, from 4.14 to 4.16.
-* One minor version to the next in sequence, for example, from 4.15 to 4.16.
-* Patch updates are also supported from z-stream to z-stream, for example 4.16.1 to 4.16.2.
+* One minor version to the next in sequence, for example, from 4.16 to 4.17.
+* Patch updates are also supported from z-stream to z-stream, for example 4.17.1 to 4.17.2.
```

Comment:
This is likely the way it has always been done, but this makes me think the 4.17.1 and 4.17.2 versions are available already.

Reply:
The versions mentioned are an example. So can be retained as is.